front_end/src/actions/bookingActions.js

```diff
@@ -0,0 +1,38 @@
+import { SET_BOOKING, SAVE_BOOKING } from "./types";
+import axios from "axios";
+
+// submit booking to backend so it can be stored in DB and reused in history page
+// then receive (res) from backend a confirmation incl all data, or an error
+export const submitBooking = newBooking => dispatch => {
+  axios
+    .post("/api/booking/confirm", newBooking)
+    .then(res => {
+      // if (res.status === 409) {
```

Comment:
Unneeded code?

Reply:
We have to uncomment (and maybe change) the code later, when Caffrey fixed the backend.